src/Components/Shifting/ShiftDetails.tsx

```diff
@@ -418,7 +442,7 @@ export default function ShiftDetails(props: { id: string }) {
               <span className="font-semibold leading-relaxed">
                 Date of Admission:{" "}
               </span>
-              {moment(consultation.created_date).format("LL") || "-"}
+              {moment(consultation.admission_date).format("LL") || "-"}
```

Comment:
`admission_date` is optional on the db. Please use `created_date` when admission date is not present.

Reply:
So I should do like this consultation.admission_date || consultation.created_date || "-" ?

Reply:
`consultation.admission_date || consultation.created_date`

Created date will be avilable in all scenarios

Reply:
Changes done @bodhish @gigincg. Review please